modules/hcp-shared-infra.adoc

```diff
@@ -11,4 +11,4 @@ Sharing infrastructure can be beneficial in constrained environments, such as in
 
 Before you share infrastructure, ensure that your infrastructure has enough resources to support {hcp}. On the {product-title} management cluster, nothing else can be deployed except {hcp}. Ensure that the management cluster has enough CPU, memory, storage, and network resources to handle the combined load of the hosted clusters. For more information, see "Sizing guidance for {hcp}".
 
-Workload must not be demanding, and it must fall within a low queries-per-second (QPS) profile. To maintain a reasonable risk profile, you can share up to 3 hosted clusters.
\ No newline at end of file
+Workload must not be demanding, and it must fall within a low queries-per-second (QPS) profile. For information about maintaining a reasonable risk profile, see "Sizing guidance for {hcp}".
```

Comment:
Seems okay here, but it does say nearly this same exact thing in the paragraph above. Just wondering if it would be better to condense and only show one line at the end pointing them to this link for more information?

Reply:
Good suggestion! I'll try to condense the two statements.